js/package.json

```diff
@@ -21,7 +21,7 @@
     },
     "dependencies": {
         "botbuilder": "^4.22.1",
-        "openai": "4.28.4"
+        "@azure/openai-assistants": "1.0.0-beta.5"
```

Comment:
Why is openai being removed? It's a peer dependency for teams-ai package and therefore required here and in the samples.

Reply:
fixed